src/rviz/default_plugin/image_display.cpp

```diff
@@ -241,6 +251,19 @@ void ImageDisplay::processMessage(const sensor_msgs::Image::ConstPtr& msg)
   texture_.addMessage(msg);
 }
 
+void ImageDisplay::setTopic(const QString& topic, const QString& datatype)
+{
+  ImageDisplayBase::setTopic(topic, datatype);
+  mouse_click->setTopic(topic);
+}
+
```

Comment:
There is no need to override `ImageDisplayBase::setTopic()` as this will call `updateTopic()` anyway.

Reply:
OK, I will remove this override.